ontology.ttl

```diff
@@ -193,6 +193,17 @@ foaf:logo rdf:type owl:AnnotationProperty .
 #    Data properties
 #################################################################
 
+###  http://odahub.io/ontology#memory
+:memory rdf:type owl:DatatypeProperty ,
```

Comment:
it's a bit more useful to have memory per process, could you do that?

Reply:
I made the corresponding change in the ontology